setup.py

```diff
@@ -56,8 +56,10 @@
     'aodncore>=1.0.0',
     'aodntools>=1.0.0',
     'cc-plugin-imos>=1.3.0',
+    'fiona>=1.8.8',
     'matplotlib>=1.5.1',
-    'fiona>=1.8.8'
+    'pillow',  # indirectly used by matplotlib in savefig. See soop_xbt_dm handler
+    'python-magic'
```

Comment:
If you're only using this in the unittests, it should go under `TESTS_REQUIRE`, below.

Reply:
i removed this because it's going to be in the new core, but yeah you're right